src/EVM.hs

```diff
@@ -161,7 +148,26 @@ makeVm o = do
       , baseState = o.baseState
       }
     , symbolic = o.symbolic
+    , forks = Seq.singleton (ForkState env block cache "")
+    , currentFork = 0
     }
+    where
+    env = Env
+      { chainId = o.chainId
+      , contracts = Map.fromList ((o.address,o.contract):o.otherContracts)
+      , freshAddresses = 0
+      }
+    block = Block
+      { coinbase = o.coinbase
+      , timestamp = o.timestamp
+      , number = o.number
+      , prevRandao = o.prevRandao
+      , maxCodeSize = o.maxCodeSize
+      , gaslimit = o.blockGaslimit
+      , baseFee = o.baseFee
+      , schedule = o.schedule
+      }
+    cache = Cache mempty mempty
```

Comment:
Is this refactoring necessary? I don't want to change code unless we need to or we should... I am not amazing at Haskell refactoring but I'm not sure this is necessary? I may be wrong...